dotCMS/src/main/java/com/dotcms/workflow/helper/WorkflowHelper.java

```diff
@@ -260,6 +260,24 @@ public WorkflowStep deleteAction(final String actionId,
         return schemes;
     } // findSchemesByContentType.
 
+    public void saveSchemesByContentType(final String contentTypeId, User user, final List<String> workflowIds) {
+
+        final ContentTypeAPI contentTypeAPI = APILocator.getContentTypeAPI(user);
+
+        try {
+
+            Logger.debug(this, "Getting the schemes by content type: " + contentTypeId);
```

Comment:
The comment could be wrong, you are saving not getting and probable want to log the workflowIds too